Sources/MapboxDirections/RouteResponse.swift

```diff
@@ -9,20 +9,47 @@ public enum ResponseOptions {
     case match(MatchOptions)
 }
 
+/**
+ A `RouteResponse` object is a structure that corresponds to a directions response returned by the Mapbox Directions API.
+ */
 public struct RouteResponse: ForeignMemberContainer {
+    /**
+     Unrecognized properties that are documented as "beta" properties in the Mapbox Directions API response.
+     */
     public var foreignMembers: JSONObject = [:]
     
+    /**
+     The raw HTTP response from the Directions API.
+     */
     public let httpResponse: HTTPURLResponse?
     
+    /**
+     The response identifier used to request the route.
+     */
     public let identifier: String?
+    
+    /**
+     An array of `Route` objects ordered by descending recommendation rank. This property may contain at most two `Route`s.
+     */
```

Comment:
```suggestion
    /**
     An array of `Route` objects sorted from most recommended to least recommended. A route may be highly recommended based on characteristics such as expected travel time or distance. This property contains a maximum of two `Route`s.
     */
```

Reply:
The API intentionally includes an array instead of two separate properties so that it can expand to multiple routes in the future. I’m not sure if we should even say how many would be returned.

```suggestion
    /**
     An array of `Route` objects sorted from most recommended to least recommended.
     
     If `RouteOptions.includesAlternativeRoutes` is set to `true`, this property contains more than one `Route` object. A route may be highly recommended based on factors such as the expected travel time or distance.
     */
```

Reply:
mapbox/mapbox-navigation-ios#1372 demonstrates that the Directions API at one point returned up to three routes.